astropy/visualization/wcsaxes/utils.py

```diff
@@ -13,7 +13,7 @@
     "transform_contour_set_inplace",
 ]
 
-MATPLOTLIB_LT_3_8 = not minversion(mpl, "3.8.dev")
+MATPLOTLIB_LT_3_8 = not minversion(mpl, "3.8")
```

Comment:
Fine not to do it in this PR, but I wonder if it would be better to centralize these kinds of limits - that way one can grep and replace whenever a minimum version is increased.